mmtf-serialization/src/main/java/org/rcsb/mmtf/serialization/quickmessagepackdeserialization/ObjectTree.java

```diff
@@ -0,0 +1,124 @@
+package org.rcsb.mmtf.serialization.quickmessagepackdeserialization;
+
+import java.util.Hashtable;
+import java.util.Map;
+
+/**
+ *
+ * @author Antonin Pavelka
+ */
+public class ObjectTree {

```

Comment:
Could we have some documentation for this class? or at least change the method names to something more usable, one letter method names without javadocs are going to be difficult to understand....